src/mongo/db/storage/wiredtiger/wiredtiger_record_store.h

```diff
@@ -277,9 +287,9 @@ class WiredTigerRecordStore : public RecordStore {
     const bool _isEphemeral;
     // True if the namespace of this record store starts with "local.oplog.", and false otherwise.
     const bool _isOplog;
-    const int64_t _cappedMaxSize;
```

Comment:
a atomic64 should be better ?
collMod is in a different thread, partial_read/write problem should happen on a 32-bit machine, some cpu-visitable problem may also be considerable